MapleServer2/Packets/ChangeBackgroundPacket.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+
+namespace MapleServer2.Packets;
+
+public static class ChangeBackgroundPacket
+{
+    public static PacketWriter ChangeBackground(string dds)
```

Comment:
Just curious, what is `dds`?

Reply:
> Just curious, what is `dds`?

Image file type (direct draw surface) used in many parts of the client, including map backgrounds